cvmfs/sync_item.h

```diff
@@ -169,6 +174,7 @@ class SyncItem {
   mutable EntryStat scratch_stat_;
 
   bool whiteout_;
```

Comment:
Actually, since we are anyway at this class, it would be helpful to add comments to the private members and briefly remind the reader of the union file systems concepts they indicate.